lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -282,43 +339,62 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         return !!changedProperties?.nodeType;
     }
 
-    revertChanges() {
-        this.changedProperties = {};
-        this.hasMetadataChanged = false;
-    }
-
-    cancelChanges() {
-        this.revertChanges();
-        this.loadProperties(this.node);
-    }
-
-    showGroup(group: CardViewGroup): boolean {
-        const properties = group.properties.filter((property) => !this.isEmpty(property.displayValue));
-
-        return properties.length > 0;
+    private loadProperties(node: Node) {
+        if (node) {
+            this.basicProperties$ = this.getProperties(node);
+            this.groupedProperties$ = this.contentMetadataService.getGroupedProperties(node, this.preset);
+            if (this.displayTags) {
+                this.loadTagsForNode(node.id);
+            }
+            this.loadCategoriesForNode(node.id);
+            if (!this.node.aspectNames.includes('generalclassifiable')) {
+                this.categories = [];
+                this.classifiableChangedSubject.next();
+            }
+        }
     }
 
-    ngOnDestroy() {
-        this.onDestroy$.next(true);
-        this.onDestroy$.complete();
+    private getProperties(node: Node) {
+        const properties$ = this.contentMetadataService.getBasicProperties(node);
+        const contentTypeProperty$ = this.contentMetadataService.getContentTypeProperty(node);
+        return zip(properties$, contentTypeProperty$)
+            .pipe(map(([properties, contentTypeProperty]) => {
+                const filteredProperties = contentTypeProperty.filter((property) => properties.findIndex((baseProperty) => baseProperty.key === property.key) === -1);
+                return [...properties, ...filteredProperties];
+            }));
     }
 
-    public canExpandTheCard(group: CardViewGroup): boolean {
-        return group.title === this.displayAspect;
+    private isEmpty(value: any): boolean {
```

Comment:
can you set type?

Reply:
`isEmpty` is then used to check if `CardViewItem` display value is not empty and it's set to any as well, I just moved the method to group public and private methods. To change this any we would have to change `CardViewItem` display value type and this would require some additional refactoring. We can create a Jira to investigate this